### ISSUE TITLE
Error in SSL site

### ISSUE THREAD

**Issue by informundo** [GordonLesti/Lesti_Fpc#189]
when I switch from a secure page to an unsecured, it gives me this error. It tries to load a resource from an address in a non-ssl SSL page
XMLHttpRequest cannot load https://www.navatiendas.es/skin/frontend/em0132/default/css/less/functions.less. No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'http://www.navatiendas.es' is therefore not allowed access.

Test with Lesti 1.4.1 to 1.4.5

**Comment by GordonLesti** [comment]
Hello @informundo 
sorry to ask that, but does the error also appear with disabled FPC? Cause Fpc isn't modifying the header of static files.

**Comment by informundo** [comment]
Yes. With FCP disabled does not show the error and the page  is displayed correctly

**Comment by GordonLesti** [comment]
Can you enable Fpc again? I would like to take a look again.
Just in general, I guess you have a problem with switching the protocol. For example:
```
Mixed Content: The page at 'https://www.navatiendas.es/' was loaded over HTTPS, but requested an insecure XMLHttpRequest endpoint 'http://www.navatiendas.es/ajaxblock/?data=eyJ0eXBlIjoidGFic1wvZ3JvdXAiLCJ0a…I4IiwidGVtcGxhdGUiOiJlbXRhYnNcL2dyb3VwLnBodG1sIiwiYWpheGJsb2NrIjp0cnVlfQ=='. This request has been blocked; the content must be served over HTTPS.
```

**Comment by informundo** [comment]
Ok. Now is enable. 
It does not happen every time. You may have to enter a secure page and return for example at home several times

**Comment by Quazz** [comment]
I find this happens when block html cache is enabled and you happen to load the http site first. 

Disabling html block cache solves this issue entirely for me, something is up with it in combination with LestiFPC I find.
